examples/bri-3/src/bri/workgroup/workgroups/agents/workgroups.agent.ts

```diff
@@ -45,11 +45,11 @@ export class WorkgroupAgent {
       participantIds,
     );
 
-    if (participantIds.length != bpiSubjects.length) {
+    if (participantIds.length !== bpiSubjects.length) {
       throw new NotFoundException(BPISUBJECT_NOT_FOUND_ERR_MESSAGE);
     }
 
-    bpiSubjects.map((bp) => {
+    bpiSubjects.forEach((bp) => {
```

Comment:
just curious, do we have any specific reasons to change this and lose on the immutability?